src/main/resources/com/google/api/codegen/py/transport.snip

```diff
@@ -50,6 +50,10 @@
                 channel = self.create_channel(
                     address=address,
                     credentials=credentials,
+                    options={
+                        'grpc.max_send_message_length': -1,
+                        'grpc.max_receive_message_length': -1,
+                    },
```

Comment:
This line needs to have `.items()` tacked on (`options` is expected to be a sequence of two-tuples, not a dict).